benchmarks/annulus/annulus.cc

```diff
@@ -1,8 +1,10 @@
+#include <aspect/simulator.h>
```

Comment:
Ah so these are essentially modifications to check the dynamic topography for the annulus? Then lets keep them